packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget.tsx

```diff
@@ -66,6 +66,42 @@ export class ViewerTransformWidget {
   @Prop({ mutable: true })
   public controller?: TransformController;
 
+  /**
+   * Determines whether or not the x-rotation is disabled on the widget
+   */
+  @Prop({ mutable: true })
+  public xRotationDisabled = false;
+
+  /**
+   * Determines whether or not the y-rotation is disabled on the widget
+   */
+  @Prop({ mutable: true })
+  public yRotationDisabled = false;
+
+  /**
+   * Determines whether or not the z-rotation is disabled on the widget
+   */
+  @Prop({ mutable: true })
+  public zRotationDisabled = false;
+
+  /**
+   * Determines whether or not the x-translation is disabled on the widget
+   */
+  @Prop({ mutable: true })
+  public xTranslationDisabled = false;
+
+  /**
+   * Determines whether or not the y-translation is disabled on the widget
+   */
+  @Prop({ mutable: true })
+  public yTranslationDisabled = false;
+
+  /**
+   * Determines whether or not the z-translation is disabled on the widget
+   */
+  @Prop({ mutable: true })
+  public zTranslationDisabled = false;
+
```

Comment:
Do we need to add a `@Watch()` for these props in the case that we change the values after the component has been initialized?

Reply:
Yes - we should!